tests/cases/fourslash/moveToNewFile.ts

```diff
@@ -23,6 +23,7 @@ a; y;`,
         "/y.ts":
 `import { b } from './other';
 import { p } from './a';
+
```

Comment:
This new line addition seems unnecessary

Reply:
#37941 claims it should be inserted. This file is brand new so I don’t see why it shouldn’t match our default auto-import styling, which adds a blank line between imports and following statements.